packages/block-editor/src/components/block-list/block-popover.js

```diff
@@ -155,13 +155,8 @@ function BlockPopover( {
 			__unstableSticky={ ! showEmptyBlockSideInserter }
 			__unstableSlotName="block-toolbar"
 			__unstableBoundaryParent
-			// Allow subpixel positioning for the block movement animation.
-			__unstableAllowVerticalSubpixelPosition={
-				moverDirection !== 'horizontal' && node
-			}
-			__unstableAllowHorizontalSubpixelPosition={
-				moverDirection === 'horizontal' && node
-			}
+			// Observe movement for block animations (especially horizontal).
+			__unstableObserveElement={ node }
```

Comment:
Seems like this is the same thing as `anchorRef`. I wonder if we should instead observe anchor ref when it's a DOM element. I also wonder whether this means the interval inside Popover is useless.